include/dd/NoiseFunctionality.hpp

```diff
@@ -0,0 +1,714 @@
+/*
+* This file is part of the MQT DD Package which is released under the MIT license.
+* See file README.md or go to https://www.cda.cit.tum.de/research/quantum_dd/ for more information.
+*/
+
+#ifndef DDSIM_NOISEFUNCTIONALITY_HPP
+#define DDSIM_NOISEFUNCTIONALITY_HPP
```

Comment:
use `#pragma once` instead of include guards.

Reply:
I removed it altogether as it is not needed